components/brave_news/browser/topics_fetcher.cc

```diff
@@ -109,7 +109,7 @@ void TopicsFetcher::OnFetchedTopics(
     api_request_helper::APIRequestResult result) {
   if (!result.Is2XXResponseCode()) {
     LOG(ERROR) << "Failed to get topics: " << result.error_code() << ", "
-               << result.body();
+               << result.SerializeBodyToString();
```

Comment:
Not sure if it is in general ok to put content of response to log. But that's out of scope